cellrank/external/kernels/_wot_kernel.py

```diff
@@ -149,21 +157,58 @@ def compute_initial_growth_rates(
             Maximum death rate.
         key_added
             Key in :attr:`adata` ``.obs`` where to add the estimated growth rates. If `None`, just return them.
+        kwargs
+            Keyword arguments for :func:`scanpy.tl.score_genes`. Only used when ``proliferation_key``
+            or ``apoptosis_key`` cannot be found in :attr:`adata` ``.obs``.
 
         Returns
         -------
         :class:`pandas.Series`
-            The estimated initial growth rates if ``key_added = None``, otherwise `None`.
+            The estimated initial growth rates if ``key_added=None``, otherwise `None`.
 
         Notes
         -----
-        Note that to run this method, you need to have estimated cell-level proliferation/apoptosis rates which should
-        be saved in :attr:`adata` ``.obs``. These rates are usually computed based on a gene set using a scoring
-        function like :func:`scanpy.tl.score_genes`. If you don't have access to such gene sets, don't worry,
-        you can use WOT without an estimate of initial growth rates.
+        If you don't have access to proliferation/apoptosis gene sets, you can use the ones defined in :mod:`cellrank`
+        for a specific organism. Alternatively, you can also use WOT without an estimate of initial growth rates. In
+        that case, make sure to use several iterations in
+        :meth:`cellrank.external.kernels.WOTKernel.compute_transition_matris` by increasing the `growth_iters`
+        parameter. A value around 3 works well in most cases.
+
+        The markers used here were taken from the following sources:
+
+            - human, proliferation - :cite:`tirosh:16:science`.
+            - human, apoptosis - `Hallmark Apoptosis, MSigDB <https://www.gsea-msigdb.org/gsea/msigdb/cards/HALLMARK_APOPTOSIS>`_.
+            - mouse, proliferation - :cite:`tirosh:16:nature`.
+            - mouse, apoptosis - `Hallmark P53 Pathway, MSigDB <https://www.gsea-msigdb.org/gsea/msigdb/cards/HALLMARK_P53_PATHWAY>`_.
```

Comment:
I would move this to another place - ideally, directly to the place where the gene sets are being defined. I can imagine other functions/kernels also requiring access to apoptosis /proliferation estimates, so we should make sure that we keep all the references in one common place.